### ISSUE TITLE
Tabbing between fields when editing a product variant broken

### ISSUE THREAD

**Issue by spencern** [reactioncommerce/reaction#3059]
As you can see in this video, the focus gets removed from the input field immediately after navigating to it via tab.

I think this is probably caused by the save action, but not certain.

http://recordit.co/nKSTQF1kCE

@kieckhafer or @kieckhafer care to take a look?

Steps to reproduce:
1. `release-1.5.0` branch
2. create or edit a product and use tab to navigate between input fields

**Comment by spencern** [comment]
Also seeing this server error:
```
Exception in changed observe/observeChanges callback: Error: Could not find element with id GRNBhie5NWePtp8Zu to change
    at [object Object]._.extend.changed (packages/ddp-server/livedata_server.js:188:13)
    at [object Object]._.extend.changed (packages/ddp-server/livedata_server.js:403:10)
    at [object Object]._.extend.changed (packages/ddp-server/livedata_server.js:1273:19)
    at Object.changed (server/publications/collections/products.js:395:18)
    at _CachingChangeObserver.observeChangesCallbacks.changed (packages/minimongo/local_collection.js:1315:28)
    at applyChange.changed (packages/minimongo/local_collection.js:727:27)
    at runWithEnvironment (packages/meteor.js:1188:24)
    at packages/meteor.js:1201:14
    at packages/mongo/observe_multiplex.js:182:30
    at Array.forEach (native)
    at Function._.each._.forEach (packages/underscore.js:139:11)
    at Object.task (packages/mongo/observe_multiplex.js:176:9)
    at [object Object].SQp._run (packages/meteor.js:819:16)
    at packages/meteor.js:796:12
```

**Comment by zenweasel** [comment]
I think server error is related to https://github.com/reactioncommerce/reaction/issues/2079 and usually disappears after reset or just some time.

**Comment by spencern** [comment]
I'm able to reproduce this error specifically by focusing fields, but I'm pulling `1.5.0-rc.3` as well as the current `release-1.5.0` branch to see if I can do a better job with this.

**Comment by spencern** [comment]
Actually on a fresh pull of `release-1.5.0` I can't reproduce the server error, so @zenweasel is probably correct that it may not be directly caused buy this issue.

**Comment by kieckhafer** [comment]
Looks to be something in [this piece of code](https://github.com/reactioncommerce/reaction/blob/release-1.5.0/imports/plugins/included/product-variant/containers/variantFormContainer.js#L178-L192) causing the issue.

When I comment that all out, and just log something as the only code inside `handleVariantFieldSave()`, then you can tab fine.

I've tried disabling the validation, and just running the `Meteor.call`, and the issue is still happening, which is odd because we run the exact same `Meteor.call` [here](https://github.com/reactioncommerce/reaction/blob/release-1.5.0/imports/plugins/included/product-admin/client/containers/productAdmin.js#L39-L51), on the parent product, and tabbing through the fields works just fine here.

Still working through it, but wanted to update here to get visibility. @mikemurray any thoughts?

**Comment by spencern** [comment]
@kieckhafer yes, this is only on variants (and options), and `handleVariantFieldSave` seems like a probably culprit as it seemed to be something about the callback that was triggering it, if I got disconnected from the server, tabbing worked well.

**Comment by spencern** [comment]
Resolved by #3074
